sdk/schemaregistry/azure-schemaregistry-avroencoder/samples/async_samples/encode_and_decode_event_data_message_async.py

```diff
@@ -23,6 +23,29 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
+"""
+FILE: encode_and_decode_event_data_message_async.py
+DESCRIPTION:
+    This sample demonstrates the following:
+     - Authenticating an async SchemaRegistryClient to be used by the AvroEncoder.
+     - Passing in content, schema, and EventData class to the AvroEncoder, to be returned an
+      EventData object containing encoded content and corresponding content type.
+     - Passing in an `EventData` object with `body` set to Avro-encoded content and `content_type`
+      set to corresponding content type to the AvroEncoder, to be returned the decoded content.
```

Comment:
- Passing in content......to return an EventData object?
- Passing in an EventData object...content_type set to the corresponding AvroEncoder content type, to return the decoded content?

to be returned could also work

Reply:
Since the content type is made of 2 things:
1) schema id
2) `avro/binary` -- which is the avro mime type.

I didn't know how exactly to describe this without over-describing? (ie. Thought saying `content type which has schema ID and avro mime type` was maybe too much.) I _thought_ `corresponding content type` would be enough description. Though maybe it's not. Thoughts? 🤔

Reply:
Sounds good to me!